### ISSUE TITLE
[Bug] Failing to add accounts

### ISSUE THREAD

**Issue by ahtaarra** [ccrama/Slide#3353]
Slide version(s): 6.6.2-3 (from F-droid), debug built from master (till commit e41a8f282298d1bb5af79e514cc2e12d30964ae7)
Android version(s): 10 (MIUI 12.0.6) , 7.1 (LineageOS 15.1)

### Issue:
When I enter my username/password pair(s) and successfully log in, after I allow the app access to the respective account, it opens the following view: ![Screenshot](https://user-images.githubusercontent.com/10814765/112193230-99e86e00-8bff-11eb-8ee0-70f7b11d9f45.jpg)

According to android studio, the issue most probably concerns:
>me.ccrama.redditslide.debug V/(Login.java:112): http://net.domain.name/ccrama.me/#_

This info has been collected while running a debug build I myself compiled using the `master` branch. This is not account specific (I tried it with multiple accounts) and also probably not device-specific (I observed the same behaviour on my phone running the F-droid version on A10 device, my own compiled build on that system and another debug build on another device running A7.1).

Only thing that can be done is closing the opened page and the _account was not added_. I did not face such issue some months ago.

My experience in app debugging is very limited but  #3334 might be relevant.

### Steps to reproduce:
1. Open the app
2. Select 'Add an account' option from the hamburger menu
3. Enter account credentials
4. Tap on either 'Allow' or 'Decline'
5. See the screenshot view

**Comment by Suryacyleis** [comment]
I am also experiencing the same on the 6.6.1-3 version from the F-droid

**Comment by BlindRambler** [comment]
Yeah this is still an issue on Galaxy S21+.  Like the OP stated, I too have done exactly as they have.  It really does look like a great app, but I am unable to use.  Perhaps we're doing something wrong?
